src/telemetry_util.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <format>
+#include <string_view>
+
+#include "string_util.h"
+
+extern "C" {
+#include <ngx_http.h>
+}
+
+namespace datadog::nginx {
+
+constexpr std::string_view relative_filepath(
+    std::string_view absolute_filepath) {
+  constexpr std::string_view prefix = "nginx-datadog";
+  const size_t pos = absolute_filepath.find(prefix);
+  if (pos != std::string_view::npos) return absolute_filepath.substr(pos);
+
+  const size_t sep_pos = absolute_filepath.find_first_of("/\\");
```

Comment:
nit: I think we should use `find_last_of` here. 
otherwise something like /Users/name.surname would still leak name.surname